styles/base/base.scss

```diff
@@ -0,0 +1,18 @@
+// Import relevant config.
+@import '../config/font.scss';
```

Comment:
can we keep base CSS at one place 

right now we have to file :
1. src/basic/base.scss
2. styles/base/base.scss

Reply:
@ankit-9 Yes we need only one base file. 

I have pushed this for review and then we can refactor existing code and merge with `styles/base/base.scss`.

Reply:
yes, we can go with a distinct folder for styles, js, and assets.